src/test/java/gov/di_ipv_fraud/pages/EnterYourDetailsExactlyDVLAPage.java

```diff
@@ -42,9 +40,6 @@ public EnterYourDetailsExactlyDVLAPage() {
     @FindBy(id = "expiryDate-year")
     public WebElement LicenceValidToYear;
 
-    @FindBy(id = "licenceIssuerRadio")
-    public WebElement DVLA;
-
     @FindBy(id = "licenceIssuerRadio-DVLA-label")
     public WebElement DVLALabel;
```

Comment:
Are we now only finding the label? Why Are we not finding the actually radio?

Reply:
@ElliotMurphyGDS : We are using the actual radio. I removed the above lines because this element was already available here https://github.com/alphagov/di-ipv-cri-fraud-test/blob/main/src/test/java/gov/di_ipv_fraud/pages/DrivingLicencePageObject.java
under this name 
    @FindBy(id = "licenceIssuerRadio")
    public WebElement radioBtnDVLA;
    
    So just reused that and removed the duplicate.